### PR TITLE
remove _fitDesignResolution function call in EVENT_AFTER_UPDATE

### PR DESCRIPTION
**Pull request by moshuying**
Re: # https://github.com/cocos/3d-tasks/issues/17320
https://github.com/cocos/3d-tasks/issues/17320

### Changelog

* Canvas switched to using Widgets, but forgot to clean up the previous alignment logic, resulting in duplicates. Cleanup required.

-------

### Continuous Integration

This pull request:

* [x] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [x] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by moshuying** [comment]
@cocos-robot run test cases

**Review by jareguo — APPROVED** [review]

**Review by jareguo — APPROVED** [review]

**Review by LinYunMo — APPROVED** [review]

File: cocos/2d/framework/canvas.ts [hunk 1]
```diff
@@ -134,16 +134,33 @@ export class Canvas extends RenderRoot2D {
...
         if (EDITOR) {
-            this._fitDesignResolution = (): void => {
+            this.fitDesignResolution_EDITOR = (): void => {
                 // TODO: support paddings of locked widget
                 this.node.getPosition(this._pos);
                 const nodeSize = view.getDesignResolutionSize();
-                Vec3.set(_worldPos, nodeSize.width * 0.5, nodeSize.height * 0.5, 0);
+                const trans = this.node._uiProps.uiTransformComp!;
+
+                let scaleX = this.node.scale.x;
```
Comment (SantyWang) [inline cocos/2d/framework/canvas.ts:143]: Don't and this one for now. Widgets adapting based on scale is a weird problem, and we're going to deprecate the effect of scale in widgets after this.

**Comment by moshuying** [comment]
run test cases

**Comment by moshuying** [comment]
@cocos-robot run test cases

**Pull request merged**